bin/install

```diff
@@ -234,8 +234,8 @@ install_default_npm_packages() {
 
   for name in $(cat $default_npm_packages); do
     echo -ne "\nInstalling \e[33m${name}\e[39m npm package... "
-
-    if npm install -g $name > /dev/null 2>&1; then
+    PATH="$PATH:$ASDF_INSTALL_PATH/bin" npm install -g $name > /dev/null 2>&1
```

Comment:
Now that I think about it, shouldn't the new path be put at the front of PATH so it's used first?

Reply:
That actually does make more sense, huh! +1